internal/config/client.go

```diff
@@ -66,6 +68,17 @@ func (c *Config) NewClient(ctx context.Context) (any, error) {
 	// setup a transport to handle digest
 	transport := digest.NewTransport(cast.ToString(c.PublicKey), cast.ToString(c.PrivateKey))
 
+	// proxy is only used for testing purposes
+	if c.ProxyPort != nil {
+		proxyURL, _ := url.Parse(fmt.Sprintf("http://localhost:%d", *c.ProxyPort))
+		transport.Transport = &http.Transport{
+			Proxy: http.ProxyURL(proxyURL),
+
+			//nolint:gosec // temporary workaround so client can connect with hoverfly instance
+			TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
+		}
+	}
```

Comment:
`InsecureSkipVerify` is a temporary workaround, have an open issue https://github.com/SpectoLabs/hoverfly/issues/1120 to understand why local hoverfly ca is not being picked up.

Reply:
went for the approach of using the cli directly to avoid this workaround.